src/components/ApplicationGroup/List/LoadingShimmer/LoadingShimmerList.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024. Devtron Inc.
+ */
+
+import './loadingShimmerList.scss'
+import { LoadingShimmerListType } from './types'
+
+export const LoadingShimmerList = ({ shimmerRowClassName, hideLastColumn = false }: LoadingShimmerListType) => {
+    const renderShimmer = () => <div className="shimmer-loading h-16 pt-8 pb-8" />
+
+    const renderLoadingRow = () => {
+        const rows = []
+        for (let i = 0; i < 3; i++) {
+            rows.push(
```

Comment:
a better way to render this would be 

```
const count = 3

Array.from(Array(count).keys()).map((key) => /* return the div here */)
```

fragment error can be avoided by directly return the above